### DISCUSSION TITLE
TTS数据集的制作有教程吗？

### DISCUSSION THREAD

**Discussion by warp-drive** [PaddlePaddle/PaddleSpeech#1636] [Q&A] [answered]
主要想了解 .interval 文件 .TextGrid 是需要自己写脚本生成还是通过 Praat 手工生成的？

**Comment by iftaken** [comment, marked as answer]
如果是用MFA或者其它的一些自动对齐工具，都是得到音素和对应的时间戳之后，通过脚本转成的这种格式，本质上也只是一个文本文件，按照指定的格式生成（MFA内部已经写好了，直接输出.TextGrid）；人工标注音素对齐信息，一般是用 Praat做标注工具，Praat可以直接将标注结果导出成 .interval 或者 TextGrid的格式。
